charts/tempo-distributed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.27.0
+version: 1.27.1
```

```suggestion
version: 1.28.0
```
I think minor version should be bumped since this is a new feature for metrics-generator